src/js/components/Sidebars.jsx

```diff
@@ -3,10 +3,13 @@ var React = require('react'),
     InspectorSidebar = require('./InspectorSidebar'),
     VisualSidebar = require('./VisualSidebar'),
     PipelinesSidebar = require('./PipelinesSidebar'),
+    Toolbar = require('./Tools'),
```

Comment:
Minor naming inconsistency here, I'd vote to go all "tools" or all "toolbar", no preference b/w the two